src/main/controllers/features/common/applicationConfirmationController.ts

```diff
@@ -18,7 +18,7 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
         const transactionId: string = session.getExtraData(SUBMISSION_ID)!;
 
         res.render(config.APPLICATION_CONFIRMATION, {
-            title: "Application submitted",
+            title: locales.i18nCh.resolveNamespacesKeys(lang).applicationSubmittedTitle,
```

Comment:
I think you don't need to do resolveNamespacesKeys(lang) here. getLocaleInfo in net line doing same thing.